pkg/geoipupdate/geoip_updater.go

```diff
@@ -15,14 +15,27 @@ import (
 	"github.com/pkg/errors"
 )
 
+var version = "4.8.0"
+
+// AddHeaderRoundTripper struct is used to add versioned User-Agent header to each request.
+type AddHeaderRoundTripper struct {
+	r http.RoundTripper
+}
+
+// RoundTrip is used to add versioned User-Agent header to each request.
+func (ahrt AddHeaderRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
```

Comment:
I wonder if it'd be better to add the header to the request in the two spots we create requests rather than this. Better in that I think it would be a bit simpler (not needing to wrap `Transport()` seems easier to understand at a glance), as well as if callers choose to pass in their own `*http.Client` rather than using `NewClient()` we'd still get `User-Agent` set. I think the latter is not unlikely. Possibly `NewClient()` being part of the public API isn't ideal in hindsight but it is what it is!